repo2docker/buildpacks/julia.py

```diff
@@ -22,9 +31,30 @@ def get_env(self):
         ]
 
     def get_path(self):
+        """Get path for Julia executables
+
+         Returns:
+             an ordered list of path strings. The path to the Julia
+             executable is added to the list.
+
+        """
         return super().get_path() + ['${JULIA_PATH}/bin']
 
     def get_build_scripts(self):
+        """
+        Return series of build-steps common to "ALL" Julia repositories
+
+        All scripts found here should be independent of contents of a
+        particular repository.
+
+        This sets up:
+
+        - a directory for the specified Julia version and path
```

Comment:
It installs Julia itself, and also installs IJulia (`Pkg.add` sounds like you are adding to something, but it really is installing the package)